src/main/scala/mesosphere/marathon/core/plugin/PluginDescriptor.scala

```diff
@@ -0,0 +1,12 @@
+package mesosphere.marathon.core.plugin
+
+import play.api.libs.json.JsObject
+
+case class PluginDefinition(id: String,
+                            plugin: String,
+                            implementation: String,
+                            tags: Option[Set[String]],
+                            configuration: Option[JsObject],
+                            info: Option[JsObject])
+
+case class PluginsDescriptor(plugins: Seq[PluginDefinition])
```

Comment:
I find `PluginsDescriptor` and `PluginDefinition` maybe too similar. What do you think about renaming `PluginsDescriptor` to `PluginsConfiguration` or maybe just `Plugins`?